<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Window</class>
 <widget class="QMainWindow" name="Main_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1576</width>
    <height>867</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XMAS Lights v1.0.0-alpha</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1576</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Audio_File"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Configuration"/>
    <addaction name="actionSave_Configuration"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_s1">
   <property name="windowTitle">
    <string>S1</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="Light_Strand_Widget" name="widget_s1"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>40</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEstablish_Connection"/>
   <addaction name="actionRebuild_Light_Data"/>
   <addaction name="separator"/>
   <addaction name="actionStart_Simulation"/>
   <addaction name="actionPause_Simulation"/>
   <addaction name="actionStop_Simulation"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_s2">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>S2</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="Light_Strand_Widget" name="widget_s2"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_s3">
   <property name="windowTitle">
    <string>S3</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="Light_Strand_Widget" name="widget_s3"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_s4">
   <property name="windowTitle">
    <string>S4</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="Light_Strand_Widget" name="widget_s4"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_s5">
   <property name="windowTitle">
    <string>S5</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="Light_Strand_Widget" name="widget_s5"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_s6">
   <property name="windowTitle">
    <string>S6</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="Light_Strand_Widget" name="widget_s6"/>
  </widget>
  <action name="actionOpen_Audio_File">
   <property name="text">
    <string>Open Audio File</string>
   </property>
  </action>
  <action name="actionLoad_Configuration">
   <property name="text">
    <string>Load Configuration</string>
   </property>
  </action>
  <action name="actionSave_Configuration">
   <property name="text">
    <string>Save Configuration</string>
   </property>
  </action>
  <action name="actionStart_Simulation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/play.png</normaloff>:/icons/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Start Simulation</string>
   </property>
   <property name="toolTip">
    <string>Start simulating lights</string>
   </property>
  </action>
  <action name="actionPause_Simulation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/pause.png</normaloff>:/icons/icons/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause Simulation</string>
   </property>
   <property name="toolTip">
    <string>Pause simulating lights</string>
   </property>
  </action>
  <action name="actionStop_Simulation">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/stop.png</normaloff>:/icons/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop Simulation</string>
   </property>
   <property name="toolTip">
    <string>Stop simulating lights</string>
   </property>
  </action>
  <action name="actionRebuild_Light_Data">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/hammer.png</normaloff>:/icons/icons/hammer.png</iconset>
   </property>
   <property name="text">
    <string>Rebuild Light Data</string>
   </property>
   <property name="toolTip">
    <string>Rebuild the light data values from </string>
   </property>
  </action>
  <action name="actionEstablish_Connection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/wifi.png</normaloff>:/icons/icons/wifi.png</iconset>
   </property>
   <property name="text">
    <string>Establish Connection</string>
   </property>
   <property name="toolTip">
    <string>Establish Connection to Edison</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Light_Strand_Widget</class>
   <extends>QWidget</extends>
   <header location="global">light_strand_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
